packages/amplify_core/test/logger/logger_test.dart

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
*nit*: 2022. Also the header is a different format than the others in this PR. We have been moving to the other format.